shopfloor_single_product_transfer_mobile/static/src/scenario/single_product_transfer.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright 2022 Camptocamp SA (http://www.camptocamp.com)
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+ */
+
+import {ScenarioBaseMixin} from "/shopfloor_mobile_base/static/wms/src/scenario/mixins.js";
+import {process_registry} from "/shopfloor_mobile_base/static/wms/src/services/process_registry.js";
+
+const SingleProductTransfer = {
+    mixins: [ScenarioBaseMixin],
+    template: `
+        <Screen :screen_info="screen_info">
+            <template v-slot:header>
+                <state-display-info :info="state.display_info" v-if="state.display_info"/>
+            </template>
+            <searchbar
+                v-if="state_in(['select_location', 'select_product', 'set_quantity'])"
+                v-on:found="on_scan"
+                :input_placeholder="search_input_placeholder"
+            />
+            <template v-if="state_is('select_product')">
+                <item-detail-card
```

Comment:
This needs to be displayed with the location info, ideally inside the location card:

CEP-01.001.06
Total available: 10

Reply:
It should not be in the top message box saying what to do. Maybe you changed it already?

Reply:
Yes, I will update this PR as soon as https://github.com/OCA/wms/pull/614 is done, because it's basically the same implementation.